sphinx/writers/latex.py

```diff
@@ -570,8 +573,8 @@ def __init__(self, document, builder):
             self.elements['logo'] = '\\sphinxincludegraphics{%s}\\par' % \
                                     path.basename(builder.config.latex_logo)
 
-        if builder.config.language \
-           and 'fncychap' not in builder.config.latex_elements:
+        if (builder.config.language and builder.config.language != 'ja' and
+                'fncychap' not in builder.config.latex_elements):
             # use Sonny style if any language specified
             self.elements['fncychap'] = ('\\usepackage[Sonny]{fncychap}\n'
```

Comment:
This means there are 3 types fncychap settings:
* no language: `Bjarne`
* language = 'ja': None
* else: `Sonny`

I understand about 'ja' case. But I don't understand why switching it on other cases. To be more simple our code, it would be better to make `Sonny` as a default setting.